packages/website/package.json

```diff
@@ -28,7 +28,7 @@
     "react-vis": "^1.11.2"
   },
   "peerDependencies": {
-    "react": "^16.0.0",
-    "react-dom": "^16.0.0"
+    "react": "^16.13.1",
+    "react-dom": "^16.13.1"
```

Comment:
That's a pretty high version for peerDeps. Perhaps this should actually be deps or devDeps since this package isn't to be exported?

Reply:
I can move it to `dependencies`